src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceCodeAction.cs

```diff
@@ -33,7 +33,10 @@ public override object GetOptions(CancellationToken cancellationToken)
 
         protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(object options, CancellationToken cancellationToken)
         {
-            if (options is MoveToNamespaceOptionsResult moveToNamespaceOptions && !moveToNamespaceOptions.IsCancelled)
+            // We won't get an empty target namespace from VS, but still should handle it w/o crashing.
+            if (options is MoveToNamespaceOptionsResult moveToNamespaceOptions &&
+                !moveToNamespaceOptions.IsCancelled &&
+                !string.IsNullOrEmpty(moveToNamespaceOptions.Namespace))
```

Comment:
if we shouldn't get this from VS, where would we get this from?  would it indicate a bug in that caller?  shoudl we also have an assert here to try to flow fixes back that direction?

Reply:
So the UI in VS doesn't allow "", but the underlying ChangeNamespace service might support moving to global namespace (if it's the only member declared in a namespace decl). SO if someone using this service dircetly outside of VS, it's possible they would pass in "" and I think we eventually would want to support that.

Reply:
sounds good.  is this scenario at least captured in tests that call the service directly?

Reply:
yes